pkg/resource/sensitive.go

```diff
@@ -227,7 +210,9 @@ func GetSensitiveParameters(ctx context.Context, client SecretClient, from runti
 				var sensitives []interface{}
 				for _, s := range *sel {
 					sensitive, err = client.GetSecretValue(ctx, s)
-					if err != nil {
+					if err != nil && kerrors.IsNotFound(err) {
+						sensitive = []byte("")
```

Comment:
So, we have tested/validated terraform does not complain about having sensitive as an empty secret? e.g. password validation

Reply:
That's correct. I have tested the behavior with both terraform cli (after updating the state and main.tf files) and with provider-jet-azure after consuming my change.